src/config.js

```diff
@@ -0,0 +1,91 @@
+import request from 'request';
+import promisify from './promisify';
+import ConsultantError from './consultant-error';
+import properties from './properties';
+import fetchIdentifier from './identifier';
+import {parseKey, keyApplies} from './key-parser';
+import atob from 'atob';
+
+const loadConfig = async (consulHost, prefix, service) => {
+	const prefixUri = prefix ? `${prefix}/` : '';
+	const response = await promisify(cb => request({
```

Comment:
since you are on promisifying http requests, why not make it just execute the method directly, so the syntax becomes

```javascript
const response = await promiseRequest({...params});
```